frontend/public/documentation.html

```diff
@@ -25,6 +25,13 @@ <h5 class="card-title">How to annotate a log</h5>
       </div>
     </div>
 
+    <div class="card">
+      <div class="card-body">
+        <h5 class="card-title">Demo with comments</h5>
+        <a target="_blank" rel="noopener noreferrer" href="https://www.youtube.com/watch?v=_-O7ryKCnlQ"><img src="/img/youtube-preview.jpg" alt="Demo video" style="max-width: 100%; display: inline-block;" /></a>
```

Comment:
Did you intentionally not use the embed iframe or you don't know them?

If you go to a youtube video https://www.youtube.com/watch?v=_-O7ryKCnlQ and then click the "Share" button (next to likes and dislikes) and then click "Embed", it generates an `<iframe>` for you. Which is the way youtube videos are typically displayed on websites. See e.g. here https://frostyx.cz/posts/lsp-for-vim-boomers

It's easy to adjust the `widht` and `height` properties as needed.

I'd probably do it that way but if you intentionally wanted to do it with an image and link to youtube, I am okay with that as well.

Reply:
In that case +1 :-)